### PR TITLE
add links to local tours to sample welcome

### PR DESCRIPTION
**Pull request by martenson**
screenshot 2016-11-15 16 14 57

### CODE REVIEW

**Comment by martenson** [comment]
closes https://github.com/galaxyproject/galaxy/issues/3164

File: static/welcome.html.sample [hunk 1]
```diff
@@ -1,43 +1,42 @@
...
+                <a target="_blank" href="https://wiki.galaxyproject.org/Admin/Tools/AddToolFromToolShedTutorial" class="btn btn-primary btn-lg">Installing Tools »</a>
+            </div>
             <br>
-
-            <a target="_blank" href="https://wiki.galaxyproject.org/Admin/Config" class="btn btn-primary btn-lg">Configuring Galaxy »</a>
-            <a target="_blank" href="https://wiki.galaxyproject.org/Admin/Tools/AddToolFromToolShedTutorial" class="btn btn-primary btn-lg">Installing Tools »</a>
-
+            <div class="container">
+                <p>Take an interactive tour:
+                    <a target="_parent" href="/tours/core.galaxy_ui" class="btn btn-default btn-xs">Galaxy UI</a>
```
Comment (dannon) [inline static/welcome.html.sample]: Drop the leading slash, maybe?  Otherwise I think this fails when there's a prefix, right?

**Comment by bgruening** [comment]
@martenson this is how we did it in Galaxy Docker flavors.

galaxy_docker_tour

Code: https://github.com/bgruening/docker-galaxy-stable/blob/master/galaxy/welcome.html#L18

**Comment by martenson** [comment]
@bgruening yeah, and that does not work, because it will open new Galaxy in the middle Galaxy pane :)

File: static/welcome.html.sample (hunk 1 above)
Comment (martenson) [reply static/welcome.html.sample]: nice catch, fixed, thanks

**Pull request merged**